zipline/gens/sim_engine.pyx

```diff
@@ -39,36 +38,39 @@ cdef class MinuteSimulationClock:
                  trading_days,
                  market_opens,
                  market_closes,
-                 all_trading_days,
                  minute_emission=False):
         self.minute_emission = minute_emission
         self.market_opens = market_opens
         self.market_closes = market_closes
         self.trading_days = trading_days
-        self.all_trading_days = all_trading_days
         self.minutes_by_day = self.calc_minutes_by_day()
 
     @cython.boundscheck(False)
     @cython.wraparound(False)
-    cpdef market_minutes(self, np.intp_t i):
+    cdef np.ndarray[np.int64_t, ndim=1] market_minutes(self, np.intp_t i):
```

Comment:
Looks like that is because `pandas.tseries.tools.to_datetime` does not currently handle `MemoryViews`; the memory view fails the `is_list_like` condition and the fallback when no cases are met in `to_datetime` is try creating an array via `np.array([arg])` and passing that back to `convert_listlike`.

In that case, the `ndarray` should be fine.